examples/storage_ingestion/storage_ingestion_dag.py

```diff
@@ -0,0 +1,161 @@
+import datetime as dt
+from dataclasses import dataclass
+from airflow.configuration import conf
+from debussy_concert.core.phrase.protocols import PExportDataToStorageMotif
+from debussy_concert.core.motif.motif_base import MotifBase
+from debussy_concert.core.config.config_environment import ConfigEnvironment
+from debussy_concert.core.config.config_dag_parameters import ConfigDagParameters
+from debussy_concert.core.service.workflow.airflow import AirflowService
+from debussy_concert.data_ingestion.composition.base import DataIngestionBase
+from debussy_concert.data_ingestion.config.movement_parameters.time_partitioned import (
+    TimePartitionedDataIngestionMovementParameters,
+    BigQueryTimeDataPartitioning)
+from debussy_concert.data_ingestion.phrase.ingestion_to_raw_vault import IngestionSourceToRawVaultStoragePhrase
+from debussy_concert.data_ingestion.config.base import ConfigDataIngestionBase
+from debussy_concert.core.service.injection import inject_dependencies
+from debussy_framework.v3.operators.storage_to_storage import StorageToStorageOperator
+from debussy_framework.v3.hooks.storage_hook import GCSHook, S3Hook
+
+
+@dataclass(frozen=True)
+class StorageParquetDataIngestionMovementParameters(TimePartitionedDataIngestionMovementParameters):
+    source_storage_type: str  # 'gcs' or 's3'
+    source_file_uri: str
+
+
+class SourceToGcsMotif(MotifBase, PExportDataToStorageMotif):
+    def __init__(self, source_storage_hook,
+                 lake_gcp_conn_id,
+                 source_file_uri,
+                 gcs_partition,
+                 name=None):
+        super().__init__(name=name)
+        self.source_storage_hook = source_storage_hook
+        self.lake_gcs_hook = GCSHook(gcp_conn_id=lake_gcp_conn_id)
+        self.source_file_uri = source_file_uri
+        self.gcs_partition = gcs_partition
+
+    def setup(self, destination_storage_uri: str):
+        self.destination_storage_uri = destination_storage_uri
+        self.gcs_schema_uri = (f'{destination_storage_uri}/'
+                               f'{self.gcs_partition}/'
+                               )
+
+    def build(self, dag, phrase_group):
+        gcs_to_gcs_task = StorageToStorageOperator(
+            task_id='source_to_raw_vault_gcs',
+            origin_storage_hook=self.source_storage_hook,
+            origin_file_uri=self.source_file_uri,
+            destiny_storage_hook=self.lake_gcs_hook,
```

Comment:
Change destiny parameters to destination

Reply:
changed